freqtrade/exchange/exchange.py

```diff
@@ -1022,8 +1022,12 @@ def get_rate(self, pair: str, refresh: bool, side: str) -> float:
                 return rate
 
         conf_strategy = self._config.get(strat_name, {})
+        dont_use_order_book = (
+            ('use_order_book' in conf_strategy or side == "sell") and
+            conf_strategy.get('use_order_book', False)
+        )
```

Comment:
I'm not sure what you're trying to do with this.
the side has nothing to do with the "use_order_book" setting - it's valid for both ask_strategy and bid_strategy - as such - i don't see why side should make this kind of a difference to this method.



If that were really the case (hint: it isn't), then please revert the whole PR that pulled these 2 methods 
apart.

You can surely simplify `if conf_strategy.get('use_order_book', False) and ('use_order_book' in conf_strategy):` to `if conf_strategy.get('use_order_book', False):` (that 2nd part is irrelevant, as `.get` will use the default if it's not in conf_strategy - but `use_order_book` is perfectly fine on both buy and sell sides.